mmengine/config/config.py

```diff
@@ -809,7 +810,16 @@ def _format_basic_types(k, v, use_mapping=False):
             if isinstance(v, str):
                 v_str = repr(v)
             else:
-                v_str = str(v)
+                try:
+                    ast.parse(str(v))
+                except SyntaxError:
+                    v_str = repr(str(v))
+                    print_log(
+                        f'Cannot parse the value: {v} of key "{k}"',
+                        logger='current',
+                        level=logging.WARNING)
```

Comment:
When will this be triggered? Should there be too many warning logs?

Reply:
This enhancement will be done in the new config design. I will close this PR.